opendps/pastunits.h

```diff
@@ -46,6 +46,7 @@ typedef enum {
     past_VIN_ADC_K,
     past_VIN_ADC_C,
     past_tft_brightness,
+    past_UPDATE_INTERVAL,
```

Comment:
This enum confused me initially when I was dealing with past_tft_brightness since I'm used to enums being uppercased. The coefficient values are also in uppercase so there isn't really any consistency.

Would it be better to uppercase or lowercase everything?

Reply:
The ADC/DAC coefficients should really be changed to const variables. This is more a call for @kanflo though as I usually leave enums as uppercase as well